o2.sh

```diff
@@ -89,7 +90,9 @@ esac
 if [[ $ALIBUILD_O2_TESTS ]]; then
   CXXFLAGS="${CXXFLAGS} -Werror"
 fi
+
 cmake $SOURCEDIR -DCMAKE_INSTALL_PREFIX=$INSTALLROOT                                        \
+      ${CMAKE_GENERATOR:+-G "$CMAKE_GENERATOR"}                                             \
```

Comment:
👍 this allows us to set it with some defaults (your example uses an ext. variable which is frowned upon, but it's the same)

Reply:
my intent was, once tested a bit, to put CMAKE_GENERATOR in the `env` part of the default-o2*  
will that be ok or do I miss something ?

Reply:
It's OK